Update README.md (#601)
DevExpress/AjaxControlToolkit: README.md

# ASP.NET AJAX Control Toolkit (for ASP.NET 4.x)

[![Build status](https://ci.appveyor.com/api/projects/status/7ad3rsrupu40mx1r/branch/master?svg=true)](https://ci.appveyor.com/project/dxrobot/ajaxcontroltoolkit/branch/master)
[![Installer](https://img.shields.io/badge/get%20installer-20.1.0-orange.svg)](https://go.devexpress.com/AjaxControlToolkit_Website_Download.aspx)

> The ASP.NET AJAX Control Toolkit is an open source library for web development maintained by DevExpress. This library contains over 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark.

Reviewer: ```suggestion
The ASP.NET AJAX Control Toolkit is an open-source library for web development maintained by DevExpress. This library contains over 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark.
```

> # ASP.NET AJAX Control Toolkit

Reviewer 2: ```suggestion
# ASP.NET AJAX Control Toolkit (for ASP.NET 4.x)
```

[![Build status](https://ci.appveyor.com/api/projects/status/7ad3rsrupu40mx1r/branch/master?svg=true)](https://ci.appveyor.com/project/dxrobot/ajaxcontroltoolkit/branch/master)
[![Installer](https://img.shields.io/badge/get%20installer-20.1.0-orange.svg)](https://go.devexpress.com/AjaxControlToolkit_Website_Download.aspx)

> The ASP.NET AJAX Control Toolkit is an open source library for web development maintained by DevExpress. This library contains more than 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark. Use these controls to create highly responsive and interactive web applications.

Reviewer 2: ```suggestion
The ASP.NET AJAX Control Toolkit is an open source library for web development maintained by DevExpress. This library contains over 40 controls, including AutoComplete, CollapsiblePanel, ColorPicker, MaskedEdit, Calendar, Accordion, and Watermark.
```

> ![logo](https://github.com/DevExpress/AjaxControlToolkit/raw/wiki-static-resources/readme-banner.png)

Reviewer 2: Let's remove that image. 

```suggestion

```

> Download the DevExpress Edition of the AJAX Control Toolkit today, evaluate it in your project and let us know what you think.  

Reviewer 2: ```suggestion
Use the following link to download AJAX Control Toolkit:  
```

> [Get the new installer from DevExpress.com](https://go.devexpress.com/AjaxControlToolkit_GitHub_DevExpressPage.aspx)

Reviewer 2: ```suggestion
[Download the Installer from DevExpress.com](https://go.devexpress.com/AjaxControlToolkit_GitHub_DevExpressPage.aspx)
```
